docs/trustedsetup.md

```diff
@@ -10,28 +10,14 @@ to manage a large amount of value, and if their users accept that the risk of
 coordinator misbehaviour is insufficient to justify doing the work nof a
 trusted setup. After all, MACI's security model presumes a trusted coordinator.
 
+The PSE team is making available a trusted set of zKeys for MACI's circuits,
+which will be available as soon as the first ceremony is completed and
+will be visible on
+[p0tion's website](https://ceremony.pse.dev/projects/Maci%20v1%20Trusted%20Setup%20Ceremony).
+For more info on the ceremony please refer to p0tion's [docs](https://p0tion.super.site/).
```

Comment:
> A project might want to run their own trusted setup for a different param combination. what you think of that?

I agree with this @ctrlc03. But the purpose of p0tion needs to be explicitly defined. It's ambiguous whether p0tion serves as an archive page displaying the results of the ceremony, a page where one can view the ongoing status, or a tool designed for users to easily conduct their own ceremonies with desired parameters. This distinction should be made clear in this document.